build/dockerfiles/Dockerfile

```diff
@@ -41,7 +41,7 @@ RUN apk --update --no-cache add \
 WORKDIR /che-machine-exec/
 COPY . .
 # to test FIPS compliance, run https://github.com/openshift/check-payload#scan-a-container-or-operator-image against a built image
-ENV CGO_ENABLED=1
+ENV CGO_ENABLED=0
```

Comment:
Should we keep `1` ?

Reply:
let's keep it 1

Reply:
@tolusha 
FYI 
AFAIK it works with `CGO_ENABLED=0` for both `ubi8` and `ubi9` based assemblies
there is a problem with `CGO_ENABLED=1`, please see https://issues.redhat.com/browse/CRW-8248?focusedId=26649924&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-26649924

Reply:
The reference in the comment above explains the reason for those errors